aerogear/mobile-security-service
# feat(auth): added a redirect before loading routes if not logged in (AEROGEAR-9072) (#154)feat(auth): added a redirect before loading routes if not logged in (AEROGEAR-9072) (#154)feat(auth): added a redirect before loading routes if not logged in (AEROGEAR-9072) (#154)Merge pull request #154 from craicoverflow/AEROGEAR-9072

## Motivation

https://issues.jboss.org/browse/AEROGEAR-9072

## What


Added an automatic redirect to the login screen if a user is not logged in.

## Why

This prevents anonymous users from gaining access to the UI. Even though the data would not return, it is still best practice.

## How
 
Converted `App.js` from a stateless component to a stateful one, connected the Redux store. Added a check in the `App.js` component and redirect to login screen when the user check fails.

## Verification Steps


1. Open the community cluster instance. Contact me for the URL.
2. Ensure that the image name for the `mobile-security-service-app` is `ephelan/mobile-security-service`.
3. Login to the security service.
4. Log out and try to access the home page. You should be redirected to the login screen.

## Checklist:


## Progress

- [ ] Complete test

## Additional Notes
